app/views/dashboards/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1 class="t-display page__heading">Dashboard</h1>
+<% @title = t('.title') %>
```

Comment:
@jvanbaarsen Of cause I've checked Dashboard page after modified this. I think no problem because it looks as same format as these page (e.g. `app/views/reverse_dependencies/index.html.erb`, `app/views/rubygems/show.html.erb` and `app/views/rubygems/show_yanked.html.erb`). And instance variable of `@title` was called for `<title>` tag on [ApplicationHelper](https://github.com/rubygems/rubygems.org/blob/master/app/helpers/application_helper.rb#L4
).